src/ResourceManagement/ApiApp/ApiApp.Tests/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
i don't see your test project contains new dependencies. Could you double check whether we do need this redirect?